server/spec/cloud_registration_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'date'
+require 'spec_helper'
+require 'candlepin_scenarios'
+
+
+
+describe 'Cloud Registration' do
+
+    include CandlepinMethods
+
+    before(:each) do
+        skip("candlepin running in standalone mode") unless is_hosted?
+        skip("cloud registration is not enabled") unless @cp.is_capability_present("cloud_registration")
```

Comment:
Given that, by default, cloud authentication is set to false, and the capability is only present if that property is set to true, that means this test suite will always be skipped when Jenkins will be running spec tests. The alternatives I can think of are either turn cloud auth on by default (probably not a great idea), or, somehow setting it in candlepin.conf before we run spec tests on jenkins.

Reply:
Yup. There are a number of ways to handle this, but the critical part here is getting the tests available. We can sort out the details in making them autorun in our testing infrastructure later.

On that note, the easiest would probably be having Jenkins write a custom.yaml with the cloud reg setting enabled. That way we don't need to do much on the infrastructure side to get it going.